dashboard/src/components/TreeListingPage/TreeListingPage.tsx

```diff
@@ -79,6 +83,13 @@ const TreeListingPage = ({ inputFilter }: ITreeListingPage): JSX.Element => {
     return <div>Error: {error.message}</div>;
   }
 
+  if (isLoading)
+    return (
+      <Skeleton className="grid h-[400px] place-items-center">
```

Comment:
since all the Skeletons are using the same classes, you could add this to a component

the way we are doing this is that components from the `ui` folder are straight from `shadcn` 
we don't wan't to touch those, we reimport in another folder and add the classes

something like
```typescript
import type { ReactNode } from 'react'
import { Skeleton as SkeletonComponent} from '@/components/ui/skeleton'; //to avoid name conflict we rename the import

type SkeletonProps = {
  children: ReactNode
}
const Skeleton = ({ children }: SkeletonProps) =>  {
<SkeletonComponent className="grid h-[400px] place-items-center"> 
  {children}
</SkeletonComponent>
}

export default Skeleton // default exports are terrible but we are using here because every project use them for components, feel free to not use it
```

and now to use you just import Skeleton from this file and wrap without the classes